tools/automation.py

```diff
@@ -134,7 +134,7 @@ def get_mentorship_type(mentorship_type_str):
     mentorship_type = mentorship_type_str.lower()
 
     if TYPE_AD_HOC in mentorship_type:
-        return TYPE_AD_HOC.replace(' ', '-')
+        return TYPE_AD_HOC
```

Comment:
please create unit test for that function :)

Reply:
I will prepare a test and add both options.

Reply:
Please do it in another pr, I will merge this one